### DISCUSSION TITLE
Is there a documentation/feature to visualize training with Tensorboard or Visdom?

### DISCUSSION THREAD

**Discussion by abhaydoke09** [facebookresearch/detectron2#2732] [Q&A]
## 📚 Documentation

I am looking for a way to visualize my training metrics such as training/validation losses, training/validation accuracies etc. Is there a way to integrate either Tensorboard or Visdom with detectron2?

**Comment by ppwwyyxx** [comment]
The official getting started tutorial has tensorboard visualization: https://colab.research.google.com/drive/16jcaJoc6bCFAQ96jDe2HwtXj7BMD_-m5

**Comment by KnightOfTheMoonlight** [comment]
@abhaydoke09 I have shared my solution to visualize by visdom. Here is my repo [https://github.com/KnightOfTheMoonlight/visdom4detectron2](https://github.com/KnightOfTheMoonlight/visdom4detectron2)
